### PR TITLE
Fixed ManyToMany warning.

### PR DESCRIPTION
**Pull request by mefowle**
Minor edit.

### CODE REVIEW

**Comment by yushan87** [comment]
Also, you introduced a new line in `tutor:admin`.

**Comment by mefowle** [comment]
Resolved the new line issue. When I merged the master I had to resolve the conflict and did not realize that happened.

File: tutor/admin.py [hunk 1]
```diff
@@ -3,4 +3,4 @@
 the model, the data will be accessible through Django's admin functionality.
 """
 
-# Register your models here.
+# Register your models here.
```
Comment (yushan87) [inline tutor/admin.py:6]: There needs to be a newline after this comment. Also, make sure that you use Unix-style line endings (`LF`) and not Windows (`CRLF`). If you are using PyCharm, the bottom row of the main window should allow you to change from Windows style to Unix style.

Comment (mefowle) [reply tutor/admin.py:6]: I almost did that, but when I looked at the code here it didn't appear to have a new line. PyCharm says it's using LF. Is there something else I need to do?

**Pull request merged**